### PR TITLE
Add SDK and CLI example for OpenAPI/Swagger

### PR DESCRIPTION
**Pull request by xanwal**
# Description

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by shohei1029 — CHANGES_REQUESTED** [review]

**Comment by shohei1029** [comment]
The runner for SDK (sdk-endpoints-online-managed-online-endpoints-openapi) seems to be failed.

File: sdk/python/endpoints/online/managed/online-endpoints-openapi.ipynb [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+    "print(res.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Create Deployment: Custom Swagger\n",
+    "\n",
+    "Custom swagger files can be integrated by inclduing them at the root of the `code` directory. The custom file should be named `swagger<version>.json` i.e. `swagger2.json`. "
```
Comment (shohei1029) [inline sdk/python/endpoints/online/managed/online-endpoints-openapi.ipynb:340]: typo

File: sdk/python/endpoints/online/managed/online-endpoints-openapi.ipynb [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use OpenAPI with Managed Online Endpoints\n",
+    "\n",
+    "This example demonstrates how to work with with OpenAPI and Managed Online Endpoints using both automatically-generated and custom Swagger files. \n",
+    "\n",
+    "The AzureML Inference Server automatically generates swagger files for scoring scripts that use [Inference Schema](https://github.com/Azure/InferenceSchema). In this example, a simple Inference Schema-decorated [scoring script](../../../../../cli/endpoints/online/managed/openapi/decorated/code/score.py) is used. For more complex examples, refer to the [Inference Schema example](../inference-schema).  "
```
Comment (shohei1029) [inline sdk/python/endpoints/online/managed/online-endpoints-openapi.ipynb:11]: Is the link to [Inference Schema example](../inference-schema) broken?

**Review by shohei1029 — APPROVED** [review]

**Review by dem108 — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**